packages/environment-check/src/checks/get-installed.ts

```diff
@@ -159,6 +160,12 @@ export async function getFioriGenVersion(): Promise<string> {
  * @returns modules and versions
  */
 export async function getProcessVersions(): Promise<NodeJS.ProcessVersions> {
-    const output = await spawnCommand('node', ['-p', 'JSON.stringify(process.versions)']);
-    return JSON.parse(output);
+    const logger = getLogger();
```

Comment:
```suggestion
```
Sorry @cianmSAP, perhaps I phrased it not precisely. Whenever you do `getLogger()`, you get a new instance where you would need to do `logger.getMessages()` and return the messages. In this case, I would just pass the logger from the consumer here:
https://github.com/SAP/open-ux-tools/blob/87138cd93b7ea7f7a3f46cdb10547133720dcec1/packages/environment-check/src/checks/environment.ts#L25-L26

like 
```typescript
const logger = getLogger();
const processVersions = await getProcessVersions(logger);
```

the logger can be an optional parameter. In case of an error we use the passed logger instance (if set) to register the error. See other proposal in `catch`

Reply:
Added